__tests__/pages/client/index.test.tsx

```diff
@@ -53,21 +72,13 @@ describe('Client', () => {
       value: { search: '?ticketNo=t123' },
     });
 
-    const cats = [
-      new ServiceCategory({ id: 'CAT101', name: 'PET ADOPTION' }),
-      new ServiceCategory({ id: 'CAT102', name: 'Pet Fostering' })]
-    const tix = new ClientTicket({});
+    const tix: ServiceRequestType = {} as unknown as ServiceRequestType;
 
     const tixSpy = jest.spyOn(clientService, 'getTicket')
```

Comment:
Same as above, in this case we don't care about the underlying functionality since we mock a return value anyway.

Reply:
thx